docs/user/intro/vitepress.rst

```diff
@@ -0,0 +1,83 @@
+VitePress
+=========
+
+.. meta::
+   :description lang=en: Learn how to host VitePress documentation on Read the Docs.
+
+`VitePress`_ is a static site generator with a focus on performance and simplicity.
+
+Minimal configuration is required to build an existing VitePress project on Read the Docs.
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+    version: 2
+
+    build:
+    os: ubuntu-lts-latest
+    tools:
+        nodejs: "latest"
+    jobs:
+        install:
+        - npm install vitepress
+        build:
+        # The site was created by running `vitepress init`
+        # and following the official guide
+        # https://vitepress.dev/guide/getting-started
+        - vitepress build docs
+        - mkdir -p $READTHEDOCS_OUTPUT/
+        - mv docs/.vitepress/dist $READTHEDOCS_OUTPUT/html
+
+.. _VitePress: https://vitepress.dev/
+
+.. button-link:: https://dbtoolsbundle.readthedocs.io/en/stable/
+   :color: secondary
+
+   🔗 DbToolsBundle uses VitePress and Read the Docs
```

Comment:
I don't like how this button renders. I think it's too big and distracting just for an example:

![Screenshot_2025-01-23_08-44-10](https://github.com/user-attachments/assets/c2873a04-abb9-49bc-b2de-02b99095cb59)

I would probably use just a sentence with a link, or a `.. note::` / tip with that sentence.